src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,11 +5,34 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private String excavatorModel;
+    private int bucketExcavatorCapacity;
```

Comment:
Don't forget about access modifiers for your custom fields in `Excavator`. You should never want to expose the object fields directly. They should be accessed through special methods (getters and setters).